price-feeder/config/currency_provider_tracker.go

```diff
@@ -0,0 +1,175 @@
+package config
```

Comment:
I think `config` might be the wrong package here, maybe we create a new folder for this file? maybe we put it in a utils package? wdyt?

Reply:
I'm a little torn on this one, it technically is a validation for your config...this file has a good structure and it wouldn't bother me if it lived here until it becomes larger and we move it to its own package.

Reply:
I agree with @zarazan on this since it seems like this isn't big enough to warrant its own package. That being said, if we plan on expanding it to use more than CoinGecko as a data source, that may justify moving it. Do you think its fine under config for now @adamewozniak?

Reply:
Agree, no need to put it in a separate package until we add more APIs 🔥